### PR TITLE
[Portuguese] Translate locales

### PR DESCRIPTION
**Pull request by roziscoding**

### CODE REVIEW

**Review by kr4t0z — CHANGES_REQUESTED** [review]
besides adjustments, LGTM

File: site/docs/.vitepress/configs/locales/pt.ts [hunk 1]
```diff
@@ -0,0 +1,526 @@
...
+        },
+      },
+    },
+  },
+};
+
+export const notFoundPt: Record<string, NotFound> = {
+  root: {
+    title: "PÁGINA NÃO ENCONTRADA",
+    backToHome: "Me leve pra casa",
```
[the remark this answers is not in the document]
Comment (roziscoding) [reply site/docs/.vitepress/configs/locales/pt.ts:516]: True. Do you think we can still keep the pun with the song in the last message tho?

**Review by roj1512 — COMMENTED** [review]
How about changing the language code to pt-br or something like that?

**Comment by roziscoding** [comment]
> How about changing the language code to pt-br or something like that?

We've discussed this, but since we're unlikely do have a pt-pt translation (just because the pt-br one should work well for pt-pt speakers) and the other files and folders don't specify a country, we decided to keep it consistent

**Comment by roj1512** [comment]
You mean Portuguese speakers in Portugal will understand your translations?

**Comment by roziscoding** [comment]
> You mean Portuguese speakers in Portugal will understand your translations?

Yes. The two variants are pretty much interchangeable, it comes to personal preference or, in this case, availability of translators

**Pull request merged**